### ISSUE TITLE
Window controls layout does not apply to non-native apps anymore

### ISSUE THREAD

**Issue by vercore** [elementary-tweaks/elementary-tweaks#122]
After today's system updates the window control layout does not apply to non-native apps anymore. At least for me (see below).

![2020_01_08_bug_report](https://user-images.githubusercontent.com/34320710/71999076-0ff1f680-3241-11ea-8983-fa1a05e8a220.png)

**Comment by ryonakano** [comment]
I suppose this is due to some changes in Gala side. Gala 3.2.0 was just released a couple of days ago and on the same day essentially the same issue with this issue was posted at elementary/gala#678.

**Comment by LucesEnBabel** [comment]
Apparently Tweaks must change both `org.pantheon.desktop.gala.appearance.button-layout` and `org.gnome.desktop.wm.preferences.button-layout`.

I did it manually with dconf-editor and it worked.

[Source](https://www.reddit.com/r/elementaryos/comments/emwhfn/window_control_layouts_not_working_on_some_apps/fdrprc2/).

**Comment by inercia** [comment]
Yes, it seems I must use these two commands for having the `:minimize,maximize,close` buttons:

```
gsettings set org.pantheon.desktop.gala.appearance button-layout :minimize,maximize,close
gsettings set org.gnome.desktop.wm.preferences button-layout :minimize,maximize,close
```

**Comment by ryonakano** [comment]
OK, I am still not sure what triggered this issue (Gala or anything else), but I made a PR that should fix it on elementary Tweaks's side: #124.
